python/thrift/appengine-ssl-gateway/.editorconfig

```diff
@@ -0,0 +1,19 @@
+# For more info, see http://EditorConfig.org
```

Comment:
We'll be enforcing style soon via other means, at least for Java.  Can you remove it from this PR, and do a separate one at the root.  I'll spend some time looking at it to see if we want to change things at all.